nuspec/Cirrious.FluentLayout.nuspec

```diff
@@ -1,19 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-    <metadata>
+	<metadata>
+        <version>2.0.1</version>
         <id>Cirrious.FluentLayout</id>
-        <version>2.0.0</version>
-        <authors>slodge</authors>
+        <title>SQLite.Net PCL - Win32 Platform</title>
+        <authors>slodge,softlion</authors>
```

Comment:
We should decide what the right approach is here. When I put together the NuGet package I left it at just @slodge since it's still primarily his project, despite others having contributed. Should we leave it at that for now, or move it and credit it to some GH organization (Mvx?), or....?